lib/streamlit/caching/memo_decorator.py

```diff
@@ -456,15 +477,15 @@ def _read_from_disk_cache(self, key: str) -> bytes:
             _LOGGER.error(e)
             raise CacheError("Unable to read from cache") from e
 
-    def _write_to_mem_cache(self, key: str, pickled_value: bytes) -> None:
+    def _write_to_mem_cache(self, key: str, value: bytes) -> None:
```

Comment:
This is a pretty pointless rename, the aftermath of a series of refactors as I tried to figure out what types these things should interact via